content/tracing/product_specs/metrics_namespace.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Traces metrics namespace
+kind: documentation
+---
+
+The trace metrics namespace is `trace.<name>.<metrics>{<tags>}` where
+
+* `<name>` is coming from the name of the integration ("redis", "pylons", etc.)
+* `<metrics>` is about the hits, errors or latency ("request.hits", etc.)
+* `<tags` the metrics are tagged by service & resource.
+
+So for pylons it might be `trace.pylons.request.hits{service:mcnulty}`.
```

Comment:
Let's use an example that sounds more generic.  McNulty is pretty internal.

Reply:
👌

Reply:
Let's also provide many more examples to have a full coverage.
We might want to cover pXX cases too.